resources/win-chocolatey/tools/chocolateyinstall.ps1

```diff
@@ -1,10 +1,10 @@
-﻿$ErrorActionPreference = 'Stop'; # stop on all errors
-$packageName = 'yarn'
+$ErrorActionPreference = 'Stop'; # stop on all errors
+
 $packageArgs = @{
-  packageName = $packageName
+  packageName = 'yarn'
   softwareName = 'Yarn*'
   fileType = 'msi'
-  silentArgs = "/qn /norestart /l*v `"$env:TEMP\chocolatey\$($packageName)\$($packageName).MsiInstall.log`""
```

Comment:
While this solves the issue of the log output, it also disables logging at all. Upstream issue for this is https://github.com/chocolatey/choco/issues/1016

Reply:
So our options are either a giant blob of Error looking xml or nothing. I prefer nothing but we can add back the error blob.